### ISSUE TITLE
Replacing "Radio" with "Illustrations" in the top bar 

### ISSUE THREAD

**Issue by mpossee** [dailybruin/caeruleum#117]
Hello! The top bar of the main site still includes Radio - please replace that tab with Illustrations as soon as possible! Radio should still be available in the side bar 💯

**Comment by simzou** [comment]
What should the Illustrations tab link to?

**Comment by mpossee** [comment]
Hi Simon!! Ideally the illustrations tab will link to either a [Spectrum](http://dailybruin.com/category/spectrum/) - esque blog for Illo or a section similar to [Graphics](http://dailybruin.com/category/graphics/) so that all standalone art can be showcased! The Spectrum idea would need to happen before replacing the tab if we go that route, so please let me know if it would be best to decide on that and open a new issue

**Comment by zeehang** [comment]
The best way to start this would be a simple category display like how we currently handle Graphics. We would need to create a new category in Wordpress called 'Illustrations' and start tagging new illustrations uploaded to the site under that category.

Once we start uploading content under that category, we can simply add the Illustrations tab in by  going to the Wordrpess Dashboard > Appearance > Menus and editing the Top Bar (Top Bar) menu.

**Comment by simzou** [comment]
Yea a spectrum like blog can be at worst quarter+ long project if you include design, development, and maintenance. Wouldn't recommend that option unless there's a very specialized need.

A sort of in between option is to use an image gallery plugin (looks we already use / have one called [Gmedia](https://codeasily.com/portfolio/gmedia-gallery-modules/albumsgrid/)) on a page where the illustrations would be uploaded. I think it might make sense to handle graphics and other primarily visual content this way too, but I don't know if it's convenient or possible to do these things through camayak if that's still being used; someone would likely have to upload through the wordpress admin.

**Comment by simzou** [comment]
For example this is series of [how to make friends](http://dailybruin.com/?page_id=304943&preview=true) illustrations in a gallery format (hopefully y'all can see this if logged into wordpress). It looks like Gmedia can automatically be bound to a category or tag so doing doing what @zeehang suggested would make illustrations automatically show up on a page like this.
